src/py/flwr/simulation/ray_transport/ray_client_proxy.py

```diff
@@ -30,8 +31,12 @@
 )
 from flwr.common.logger import log
 from flwr.server.client_proxy import ClientProxy
+from flwr.simulation.ray_transport.ray_actor import VirtualClientEngineActorPool
 
 ClientFn = Callable[[str], ClientLike]
+ClientRes = Union[
+    common.GetPropertiesRes, common.GetParametersRes, common.FitRes, common.EvaluateRes
+]
```

Comment:
This is redundant with the definition in `ray_actor`. It should be defined in only one module, and then imported in the other.

Reply:
good point. now fixed